static/app/views/replays/details.tsx

```diff
@@ -144,6 +144,11 @@ function ReplayDetails({params: {replaySlug}}: Props) {
     );
   }
 
+  const videoEvents = replay?.getVideoEvents();
```

Comment:
There's `replay.isVideoReplay()` since we don't need the events

Reply:
ahh nice, thank you!